### PR TITLE
fix markdown and typos in teams testing

### PR DESCRIPTION
**Pull request by TessFerrandez**
# Pull Request Template

## What are you trying to address

- Write a detailed description of all changes - and if appropriate, why they are needed

## Checklist

**READY TO PR? Use the check-list below to ensure your branch is ready for PR.**

- [ ] Changes follow the repo structure and land in the appropriate folder and section
- [ ] No confidential information
- [ ] No duplicated content
- [ ] Labeled appropriately
- [ ] This PR was reviewed by at least one subject matter expert
- [ ] No lint check errors related to your changes

> Note: You may see link check errors on pages you have not touched. This is normal, and due to either broken links or sites that reject link checker bots. The reviewer will help you get to a green state on these.

### CODE REVIEW

File: docs/automated-testing/ui-testing/teams-tests.md [hunk 1]
```diff
@@ -49,72 +49,91 @@ run-tests-dev:
...
   builder.setEdgeOptions(new edge.Options().addArguments("--inprivate"));
   return builder.build();
 };
 ```
 
 ## Mobile based UI tests
-Testing your custom Teams application on mobile devices is a bit more difficult using the web-based approach as it requires usage of actual or simulated devices. Running such tests in a CI/CD pipeline can be more difficult and resource-intensive. 
 
-One approach is to use real devices or cloud-based emulators from vendors such as [BrowserStack](https://www.browserstack.com/) which requires a license. Alternatively, you can use virtual devices hosted in Azure Virtual Machines. 
+Testing your custom Teams application on mobile devices is a bit more difficult using the web-based approach as it requires usage of actual or simulated devices. Running such tests in a CI/CD pipeline can be more difficult and resource-intensive.
```
[the remark this answers is not in the document]
Comment (TessFerrandez) [reply docs/automated-testing/ui-testing/teams-tests.md:70]: I'll change it

File: docs/automated-testing/ui-testing/teams-tests.md [hunk 2]
```diff
@@ -49,72 +49,91 @@ run-tests-dev:
 ...
 ```
 
-When running test, you need to use the Selenium server URL for remote execution. With the definition from above, the URL is: http://selenium:4444/wd/hub. The code snipped below illustrates how one could initialize the Selenium driver to point to the remote Selenium server using JavaScript:
+When running test, you need to use the Selenium server URL for remote execution. With the definition from above, the URL is: `http://selenium:4444/wd/hub`.
```
Comment (adinastoll) [inline docs/automated-testing/ui-testing/teams-tests.md:52]:
```suggestion
When running a test, you need to use the Selenium server URL for remote execution. With the definition from above, the URL is: `http://selenium:4444/wd/hub`.
```

**Review by shiranr — APPROVED** [review]
LGTM

**Comment by shiranr** [comment]
@nyouens / @tompaana can one of you please approve?  :)

File: docs/automated-testing/ui-testing/teams-tests.md [hunk 3]
```diff
@@ -2,35 +2,35 @@
...
+To implement web-based UI tests for your Teams application, you can follow the same approach as for testing any other web application with a UI. [UI testing](README.md) provides good guidance on that. The starting point for your test would be to launch a browser in an automated way (using Selenium, or similar frameworks) and go to [https://teams.microsoft.com/](https://teams.microsoft.com/).
 
-If you would like to test a Teams app which was not yet published into the Teams' store, or if you would like to test the DEV/QA version of your app:
-- Using for example [Teams Toolkit](https://github.com/OfficeDev/TeamsFx), package your app based on the [manifest.json](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema). See below an example on how to achieve that:
+If you would like to test a Teams app that is not yet published into the Teams' store, or if you would like to test the DEV/QA version of your app you can use the [Teams Toolkit](https://github.com/OfficeDev/TeamsFx) and package your app based on the [manifest.json](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema)
```
Comment (tompaana) [inline docs/automated-testing/ui-testing/teams-tests.md:23]:
```suggestion
If you would like to test a Teams app that is not yet published into the Teams' store, or if you would like to test the DEV/QA version of your app you can use the [Teams Toolkit](https://github.com/OfficeDev/TeamsFx) and package your app based on the [manifest.json](https://learn.microsoft.com/microsoftteams/platform/resources/schema/manifest-schema)
```

File: docs/automated-testing/ui-testing/teams-tests.md [hunk 4]
```diff
@@ -2,35 +2,35 @@
...
 
 ```javascript
 npx teamsfx package --env dev --manifest-path ...
 ```
 
-Once the app is installed, implement [selectors](https://www.browserstack.com/guide/css-selectors-in-selenium) to [access your custom app](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-upload#access-your-app) and to perform various actions within the app. 
+Once the app is installed, implement [selectors](https://www.browserstack.com/guide/css-selectors-in-selenium) to [access your custom app](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-upload#access-your-app) and to perform various actions within the app.
```
Comment (tompaana) [inline docs/automated-testing/ui-testing/teams-tests.md:29]:
```suggestion
Once the app is installed, implement [selectors](https://www.browserstack.com/guide/css-selectors-in-selenium) to [access your custom app](https://learn.microsoft.com/microsoftteams/platform/concepts/deploy-and-publish/apps-upload#access-your-app) and to perform various actions within the app.
```

File: docs/automated-testing/ui-testing/teams-tests.md [hunk 5]
```diff
@@ -126,14 +145,14 @@ emulator -list-avds
 
 ### Configure the VM
 
-This approach essentially hosts a virtual device in a virtual machine. In order to be able to set up the emulator (Android Virtual Device) in an Azure VM, the VM should support [nested virtualization](https://azure.microsoft.com/en-us/blog/nested-virtualization-in-azure/). 
+This approach essentially hosts a virtual device in a virtual machine. In order to be able to set up the emulator (Android Virtual Device) in an Azure VM, the VM should support [nested virtualization](https://azure.microsoft.com/en-us/blog/nested-virtualization-in-azure/).
```
Comment (tompaana) [inline docs/automated-testing/ui-testing/teams-tests.md:148]:
```suggestion
This approach essentially hosts a virtual device in a virtual machine. In order to be able to set up the emulator (Android Virtual Device) in an Azure VM, the VM should support [nested virtualization](https://azure.microsoft.com/blog/nested-virtualization-in-azure/).
```

**Review by tompaana — APPROVED** [review]

**Pull request merged**